crypto/src/conversation.rs

```diff
@@ -93,6 +102,16 @@ impl MlsConversationCreationMessage {
 }
 
 impl MlsConversation {
+    /// Creates a new group/conversation
+    ///
+    /// # Arguments
+    /// * `id` - group/conversation identifier
+    /// * `author_client` - the client responsible for creating the group
+    /// * `config` - group configuration
+    /// * `backend` - the KeyStore to persiste the group
```

Comment:
As @beltram suggested earlier, the `backend` / MLS Provider isn't just the keystore. There are sometimes other operations involved (such as calling the PRNG or crypto operations).
```suggestion
    /// * `backend` - MLS Provider that will be used to persist the group
```